File: algorithm/src/main/java/baekjoon/p09325/Main.java
package baekjoon.p09325;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int t = sc.nextInt();

        while (t-- > 0) {
            int s = sc.nextInt();
            int n = sc.nextInt();
            int[][] inputs = new int[n][2];

            for (int[] input : inputs) {
                input[0] = sc.nextInt();
                input[1] = sc.nextInt();
            }

            System.out.println(solve(s, inputs));
        }
    }

    private static int solve(int s, int[][] inputs) {
        int ret = s;

        for (int[] input : inputs) {
            ret += (input[0] * input[1]);
        }

        return ret;
    }

}

File: algorithm/src/main/java/baekjoon/p09325/Main.kt
package baekjoon.p09325

fun main(args: Array<String>) {
    repeat(readLine()!!.toInt()) {
        val s = readLine()!!.toInt()
        val n = readLine()!!.toInt()
        val inputs = Array(n) { IntArray(2) }

        for (input in inputs) {
            val arr = readLine()!!.split(" ").map { it2 -> it2.toInt() }
            input[0] = arr[0]
            input[1] = arr[1]
        }

        println(solve(s, inputs))
    }
}

private fun solve(s: Int, inputs: Array<IntArray>): Int {
    var ret = s

    for (input in inputs) {
        ret += (input[0] * input[1])
    }

    return ret
}